app/jobs/regular/automatic_group_membership.rb

```diff
@@ -19,8 +19,6 @@ def execute(args)
         group.add(user, automatic: true)
         GroupActionLogger.new(Discourse.system_user, group).log_add_user_to_group(user)
       end
-
-      Group.reset_counters(group.id, :group_users)
```

Comment:
Not necessary as we're adding users via ActiveRecord which would trigger the proper callbacks.